### DISCUSSION TITLE
Warden 0.13.0 (w/ Apple Silicon images!)

### DISCUSSION THREAD

**Discussion by navarr** [wardenenv/warden#620] [Announcements]
## Dependency Requirement Change
* Warden now requires Docker Compose standalone 2+. [How to Install Docker Compose standalone](https://docs.docker.com/compose/install/other/)

## What's Changed in Warden
* Updated environment default Node version from 10 to 12 by @davidalger in https://github.com/wardenenv/warden/pull/425
* Update default URL for ElasticsearchHQ by @ihor-sviziev in https://github.com/wardenenv/warden/pull/428
* Corrected syntax of env-init.help command by @davidalger in https://github.com/wardenenv/warden/pull/490
* Add `traefik.docker.network` label to specify docker network exclusively by @tdgroot in https://github.com/wardenenv/warden/pull/458
* Require docker compose 2.2.3 when compose v2 is active by @davidalger in https://github.com/wardenenv/warden/pull/489
* Ignore the only root .git directory by @ihor-sviziev in https://github.com/wardenenv/warden/pull/496
* Allow dnsmasq service to be disabled  by @davidhiendl in https://github.com/wardenenv/warden/pull/462
* Update Magento2 defaults to 2.4.6 by @lalittmohan in https://github.com/wardenenv/warden/pull/618
* The ability to specify `DB_DISTRIBUTION=(mysql|mariadb)` and `DB_DISTRIBUTION_VERSION` by @rootindex in https://github.com/wardenenv/warden/commit/dd5ffddf4764d43c70387435c7c75035615661f0
* Add `WARDEN_OPENSEARCH=1` and `OPENSEARCH_VERSION` by @navarr in https://github.com/wardenenv/warden/commit/2bd95457748c1d639dc6109018d963ff624137ac
* Migration from Selenium to Seleniarm by @navarr in https://github.com/wardenenv/warden/commit/471dc0411771e21448dd4aa9eba8e8fdc7abdfdb
* Scoped environment config loading by @tdgroot in https://github.com/wardenenv/warden/pull/451
* Portainer made optional and off by default by @bap14 in https://github.com/wardenenv/warden/commit/03783b65cfdd644133e75468808a93e751922077
* Persist MySQL command history by @navarr in https://github.com/wardenenv/warden/commit/45a16f064680dec0e7cb3fe4906090359bd3dfd5

**Full Changelog**: https://github.com/wardenenv/warden/compare/0.12.0...0.13.0

## What's Changed in Images

To pull image changes, you can use `warden svc pull;warden svc up` for Service images and `warden env pull;warden env up` for Environment images.

* Images have moved to https://github.com/wardenenv/images
* Improved ARM64 Images Across the Board
  * Lots of work was done by @davidalger, @ihor-sviziev, @drpayyne and others
  * Recently PHP Images were updated to CentOS 9 by @navarr and now supports ARM64 🎉 
  * Core images were imported from Den with details at #574
* PHP Images support `COMPOSER_VERSION=2.2` in addition to the values of `1` and `2`.

## What's Changed in Docs
* Docs have moved to https://github.com/wardenenv/docs
* Document Windows support via WSL2 by @hardyjohnson in https://github.com/wardenenv/warden/pull/422
* Fix WSL2 paths on installations page by @ihor-sviziev in https://github.com/wardenenv/warden/pull/424
* Improve ssh connection informations. Add Mysql workbench settings by @alinalexandru in https://github.com/wardenenv/warden/pull/433
* Small docs improvement in https://github.com/wardenenv/warden/pull/472
* Elasticsearch documentation page added by @marcinsdance in https://github.com/wardenenv/warden/pull/436
* Fixing unclear docs for configuring magento 2 credentials by @Nuuttif in https://github.com/wardenenv/warden/pull/512
* Documents WordPress support by @wojtekn in https://github.com/wardenenv/warden/pull/521

## New Contributors
* @marcinsdance made their first contribution in https://github.com/wardenenv/warden/pull/436
* @Nuuttif made their first contribution in https://github.com/wardenenv/warden/pull/512
* @wojtekn made their first contribution in https://github.com/wardenenv/warden/pull/521
* @michielgerritsen made their first contribution in https://github.com/wardenenv/warden/pull/546
* @davidhiendl made their first contribution in https://github.com/wardenenv/warden/pull/462
* @rbouma made their first contribution in https://github.com/wardenenv/warden/pull/583
* @lalittmohan made their first contribution in https://github.com/wardenenv/warden/pull/618

## Our Sponsors
A huge shoutout to the number of sponsors that have come on between 0.12 and 0.13, especially our Gold sponsors:

[![SwiftOtter](https://warden.dev/img/sponsors/swiftotter.svg)](https://www.swiftotter.com/)  
[![Sansec.io](https://warden.dev/img/sponsors/sansec.svg)](https://www.sansec.io/)  
[![Hyvä](https://user-images.githubusercontent.com/145128/226427529-53483968-c9ab-484a-9ae3-c6abb58f81c9.png)](https://www.hyva.io/)

<hr /><em>This discussion was created from the release <a href='https://github.com/wardenenv/warden/releases/tag/0.13.0'>Warden 0.13.0</a>.</em>

**Comment by navarr** [comment]
Please note that as of this release, there are no longer Beta images being published.  If you are using `WARDEN_NIGHTLY=1` in your `.env` or `~/.warden/.env` you should remove it to ensure you continue pulling the latest images.

Documentation and features to ease the transition from Den are still forthcoming.

**Comment by zamoroka** [comment]
Hi @navarr 
> added backup and restore command for easy project share ...

could you please check if this was merged? Can't find commands/backup.cmd in develop and 0.13.0

**Reply by navarr** [reply]
Good call-out.  These were apparently reverted, so I've removed them from the changelist.  My apologies for the mixup.

**Reply by zamoroka** [reply]
any plans of adding this in the future?

**Reply by navarr** [reply]
I've re-added the commit and opened a PR, but its a low priority https://github.com/wardenenv/warden/pull/623
